packages/components/src/markdown-react/render.tsx

```
@@ -130,8 +128,8 @@ export class MarkdownReactRenderer {
     return text;
   }
 
-  html(html: ReactNode) {
-    return html;
+  html(text: string) {
+    return <div dangerouslySetInnerHTML={{ __html: text }} />;
```

Biome

[error] Avoid passing content using the dangerouslySetInnerHTML prop.

Setting content using code can expose users to cross-site scripting (XSS) attacks

(lint/security/noDangerouslySetInnerHtml)